libs/platform/src/lib/components/action-button-group/action-button-group.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ActionButtonGroupComponent } from './action-button-group.component';
+
+describe('ActionButtonGroupComponent', () => {
+  let component: ActionButtonGroupComponent;
+  let fixture: ComponentFixture<ActionButtonGroupComponent>;

```

Comment:
I would just add a few tests here to have coverage on the group level .

Reply:
> I would just add a few tests here to have coverage on the group level .

> I have added the test case for ActionButtonGroupComponent.

Reply:
>Fixed. I have added the test case for ActionButtonGroupComponent.